### PR TITLE
[Word] (quick start) Refresh Visual Studio version

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: docs/quickstarts/word-quickstart.md [hunk 1]
```diff
@@ -287,11 +286,11 @@ Congratulations, you've successfully created a Word task pane add-in! Next, lear
 
 1. Using Visual Studio, test the newly created Word add-in by pressing **F5** or choosing **Debug** > **Start Debugging** to launch Word with the **Show Taskpane** add-in button displayed in the ribbon. The add-in will be hosted locally on IIS.
 
-2. In Word, choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane. (If you're using the volume-licensed perpetual version of Office, instead of the Microsoft 365 version or a retail perpetual version, then custom buttons are not supported. Instead, the task pane will open immediately.)
+1. In Word, if the add-in task pane isn't already open, choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane. (If you're using the volume-licensed perpetual version of Office, instead of the Microsoft 365 version or a retail perpetual version, then custom buttons aren't supported. Instead, the task pane will open immediately.)
 
     ![Screenshot of the Word application with the Show Taskpane button highlighted.](../images/word-quickstart-addin-0.png)
```
Comment (samantharamon) [inline docs/quickstarts/word-quickstart.md:291]:
```suggestion
    ![The Word application with the Show Taskpane button highlighted.](../images/word-quickstart-addin-0.png)
```

File: docs/quickstarts/word-quickstart.md [hunk 2]
```diff
@@ -287,11 +286,11 @@ Congratulations, you've successfully created a Word task pane add-in! Next, lear
...
 
-3. In the task pane, choose any of the buttons to add boilerplate text to the document.
+1. In the task pane, choose any of the buttons to add boilerplate text to the document.
 
     ![Screenshot of the Word application with the boilerplate add-in loaded.](../images/word-quickstart-addin-1b.png)
```
Comment (samantharamon) [inline docs/quickstarts/word-quickstart.md:295]:
```suggestion
    ![The Word application with the boilerplate add-in loaded.](../images/word-quickstart-addin-1b.png)
```

File: docs/quickstarts/word-quickstart.md (hunk 1 above)
Comment (samantharamon) [reply docs/quickstarts/word-quickstart.md:291]: Consider adding a mouse cursor/pointer on the Show Taskpane button for additional visibility.

**Review by samantharamon — APPROVED** [review]
Looks good! Added a few suggestions for your consideration.

File: docs/quickstarts/word-quickstart.md [hunk 3]
```diff
@@ -265,13 +264,13 @@ Congratulations, you've successfully created a Word task pane add-in! Next, lear
...
+1. The `DefaultValue` attribute of the `DisplayName` element has a placeholder. Replace it with **My Office Add-in**.
 
-4. The `DefaultValue` attribute of the `Description` element has a placeholder. Replace it with **A task pane add-in for Word**.
+1. The `DefaultValue` attribute of the `Description` element has a placeholder. Replace it with **A task pane add-in for Word**.
 
-5. Save the file.
+1. Save the file.
 
     ```xml
     ...
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/quickstarts/word-quickstart.md:276]: That comment is found in the VS template. I'll follow up on this offline to see if we can get this update made to the source.

**Pull request merged**